### ISSUE TITLE
Cant seem to get this to work on the UDM Pro

### ISSUE THREAD

**Issue by AJEvans3** [Silvenga/unifi-udm-api-proxy#13]
I am running Docker and Docker Compose on a Synology NAS (not sure if I set up Docker compose correctly though). But I keep getting a notification on Home Assistant saying that my config.yaml is incorrect but I have no idea what I did wrong. Please help.
![Annotation 2020-03-30 164523](https://user-images.githubusercontent.com/56004016/77972804-4327ad00-72a7-11ea-828c-6aa54ef8dd29.png)
![Annotation 2020-03-30 164747](https://user-images.githubusercontent.com/56004016/77972807-4327ad00-72a7-11ea-97c2-b76415348a47.png)
![Annotation 2020-03-30 164631](https://user-images.githubusercontent.com/56004016/77972808-43c04380-72a7-11ea-94ca-79a34377908a.png)
![Annotation 2020-03-30 164613](https://user-images.githubusercontent.com/56004016/77972810-43c04380-72a7-11ea-8d39-3f94747c6a53.png)

**Comment by Silvenga** [comment]
Docker Compose? Did you install the proxy via the add-on (https://github.com/Silvenga/unifi-udm-api-proxy-ha)?

I'll need to see the logs of the proxy and the relevant Home Assistant logs.

**Comment by AJEvans3** [comment]
![Screenshot_20200330-194612_Home Assistant](https://user-images.githubusercontent.com/56004016/77981625-53e41d00-72bf-11ea-851a-3f82d09d9ff4.jpg)
![Screenshot_20200330-194621_Home Assistant](https://user-images.githubusercontent.com/56004016/77981628-547cb380-72bf-11ea-95e4-116e0513a3d2.jpg)
![Screenshot_20200330-194630_Home Assistant](https://user-images.githubusercontent.com/56004016/77981629-55154a00-72bf-11ea-8181-1de24c8e1f47.jpg)


I did not install it. I didn't even realize it was an add-on now. Wow. How do I configure it?

**Comment by AJEvans3** [comment]
I installed the Ad on but it is still not working. the logs show the same as what is shown above

**Comment by Silvenga** [comment]
I asked about the add-on because that hostname you setup only works with the add-on - it's the internal network hostname.

I need to see the the logs from the add-on, and the logs from home-assistant (not Supervisor).

**Comment by AJEvans3** [comment]
Apologies. Here are the logs from the add on:

![image](https://user-images.githubusercontent.com/56004016/78084969-6c5f4080-736e-11ea-965f-01566bcd3eca.png)

![image](https://user-images.githubusercontent.com/56004016/78084994-813bd400-736e-11ea-9584-1fb6b68fcd58.png)

**Comment by Silvenga** [comment]
Make sure to update the `udm.uri` with the IP address of the UDMP. 😹

**Comment by AJEvans3** [comment]
Wow. huge rookie move. Do I need to include the port number as well?

**Comment by Silvenga** [comment]
Not a problem 😸 

Only if the UDMP is on a non-default port - which I don't think can be configured right now.

**Comment by AJEvans3** [comment]
Im still getting the same incorrect config message. Here is the config and logs:

![image](https://user-images.githubusercontent.com/56004016/78085724-9fa2cf00-7370-11ea-9e16-b92dd4a9ddce.png)

**Comment by Silvenga** [comment]
I've never seen a 499 from UDMP - that suggests a timeout of some sort.
- Can you log into the UDMP with this account at that URL?
- Are you using a local user or a cloud user (username or email)?

**Comment by AJEvans3** [comment]
Yes, I can log into the UDMP at that URL. And I believe I am using a local user. It is an a username. Does 2FA make a difference? I have that turned on.

**Comment by Silvenga** [comment]
2FA might be it - I don't think unifiprotect supports that at all. How did you enable 2FA? I didn't know that was a feature for local accounts.

**Comment by AJEvans3** [comment]
I guess I enabled it on my unifi account in general. It also looks like my local username is an email address

**Comment by Silvenga** [comment]
I would try to setup a separate limited user for just home assistant. That's what I've seen most people use. Unfortunately this project cannot support 2FA if the client project (unifiprotect) does not also support it.

**Comment by AJEvans3** [comment]
Alright, so I made a separate limited user for just home assistant. The configuration notification has gone away but I am still unable to add the camera feeds. The add on logs are now showing the following:

![image](https://user-images.githubusercontent.com/56004016/78164059-0ae6b280-73fe-11ea-88ed-e6ec7effb021.png)

![image](https://user-images.githubusercontent.com/56004016/78164024-002c1d80-73fe-11ea-9698-23c8ab209f6a.png)

**Comment by AJEvans3** [comment]
knml;

**Comment by AJEvans3** [comment]
Ive tried uninstalling and reinstalling everything and I am still getting the safe log file results

**Comment by Silvenga** [comment]
This doesn't look to be a problem with the proxy. Did you enable RTSP for example?

**Comment by AJEvans3** [comment]
Yes, RTSP is enabled on all cameras.

**Comment by Silvenga** [comment]
I'm not sure how to help, the logs from the proxy show everything succeeding that I would expect. Maybe look at the home-assistant log around the unifiprotect extension?
